cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/Biobank.java

```diff
@@ -0,0 +1,293 @@
+        return Objects.equals(context, biobank.context)
+               && Objects.equals(getId(), biobank.getId())
+               && Objects.equals(getIdentifiers(), biobank.getIdentifiers())
+               && Objects.equals(getBiobankType(), biobank.getBiobankType())
+               && Objects.equals(getName(), biobank.getName())
+               && Objects.equals(getMainLanguage(), biobank.getMainLanguage())
+               && Objects.equals(getStoreUntilDate(), biobank.getStoreUntilDate())
+               && Objects.equals(getStartDate(), biobank.getStartDate())
+               && Objects.equals(getStatus(), biobank.getStatus())
+               && Objects.equals(getCreated(), biobank.getCreated())
+               && Objects.equals(getLastModified(), biobank.getLastModified())
+               && Objects.equals(getCoordinatingOrganization(), biobank.getCoordinatingOrganization())
+               && Objects.equals(getCoordinatingUnit(), biobank.getCoordinatingUnit())
+               && Objects.equals(getCoordinator(), biobank.getCoordinator())
+               && Objects.equals(getProject(), biobank.getProject())
+               && Objects.equals(getExternalSources(), biobank.getExternalSources())
+               && Objects.equals(getApprovals(), biobank.getApprovals())
+               && Objects.equals(getBiobankMaterials(), biobank.getBiobankMaterials());
+    }
+
+    @Override
+    @JacocoGenerated
+    public int hashCode() {
+        return Objects.hash(context, getId(), getIdentifiers(), getBiobankType(), getName(), getMainLanguage(),
+                            getStoreUntilDate(), getStartDate(), getStatus(), getCreated(), getLastModified(),
+                            getCoordinatingOrganization(), getCoordinatingUnit(), getCoordinator(),
+                            getProject(), getExternalSources(), getApprovals(), getBiobankMaterials());
+    }
+
+    @Override
+    @JacocoGenerated
+    public String toString() {
+        return this.toJsonString();
+    }
+
+    private static List<Map<String, String>> toCristinIdentifier(CristinBiobank cristinBiobank) {
+        var id = nonNull(cristinBiobank.getCristinBiobankId())
+            ? Map.of(TYPE, CRISTIN_IDENTIFIER_TYPE, VALUE, cristinBiobank.getCristinBiobankId())
+            : null;
+        var biobank = nonNull(cristinBiobank.getBiobankId())
+            ? Map.of(TYPE, "FHI-BiobankRegistry", VALUE, cristinBiobank.getBiobankId())
+            : null;
+
+        return Stream.of(id,biobank).filter(Objects::nonNull).collect(Collectors.toUnmodifiableList());
+    }
+
+    private static URI toCoordinatingOrganization(CristinOrganization cristinOrganization) {
+        return
+            Optional.ofNullable(cristinOrganization)
+                .map(CristinOrganization::getInstitution)
+                .map(CristinInstitution::getCristinInstitutionId)
+                .map(instId -> getNvaApiId(String.format("%s.0.0.0", instId), UNITS_PATH))
```

Comment:
an organization institution does not always end in 0.0.0. Look at UiO, which has toplevel 185.90.0.0. Do we have this conversion somewhere else in code? Im pretty sure we extract unit uris elsewhere

Reply:
good point!